static/src/js/components/OrderOptionList/__tests__/OrderOptionList.test.jsx

```diff
@@ -99,6 +106,7 @@ describe('OrderOptionList', () => {
     test('render a table with 2 order options', async () => {
       setup({})
 
+      expect(await screen.findByText(/You are viewing order options for the following providers:/)).toBeInTheDocument()
```

Comment:
Nitpicky here, I would also check to to see if the correct provider is showing. 

`expect(await screen.findByText(/You are viewing order options for the following providers: MMT_2/)).toBeInTheDocument()`